src/proof/alethe/alethe_printer.cpp

```diff
@@ -87,17 +128,29 @@ void AletheProofPrinter::print(
     std::shared_ptr<ProofNode> pfn,
     const std::map<Node, std::string>& assertionNames)
 {
-  Trace("alethe-printer") << "- Print proof in Alethe format. " << std::endl;
+  Trace("alethe-printer") << "- Print proof in Alethe format.\n";
```

Comment:
I find `std::endl` ugly to look at, but I always used "\n" because I assumed there was no difference. Looking more into this now, there seems to be sufficiently many small things that using `std::endl` indeed should be preferable. I'll change.